CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## [0.7.1] - 2022-04-13
+
+### Build
+
+- Implement full release workflow in the release script (#332) (#345)
+
```

Comment:
Logically, it should include all changes from dev releases, isn't?

Reply:
This is a small update to 0.7 stable branch, and the script added changes between 0.7.0 and 0.7.1. So, you have two sections:

* **0.7.1-dev.1** - with changes in that version
* **0.7.1** - with changes between releasing dev.1 and final release.

If I understand correctly, you suggest that for a full release, we should not create **0.7.1-dev.1** version, just put everything into **0.7.1** ?

Reply:
I cannot fix the script quickly, needs to be a separate task. Updated the changelog manually.